CityOfDetroit/bloom
# Feature/pfeeny listing updated on panel to sidebar

# Pull Request Template

## Issue

Addresses #80 


## Description

Created a new panel for the listing sidebar that includes the listing updated date.  This change creates the UI-component and implements it on the listing page.

## Type of change


## How Can This Be Tested/Reviewed?

Open any listing and look in the upper right corner of the sidebar.  You should see the Listing Updated panel above the Application Due panel.
![image](https://user-images.githubusercontent.com/82653098/127557537-ff84e54f-71ea-4969-9485-61617a98098a.png)



## Checklist:
